src/main/resources/openapi/schemas/reportUseOverTimeJournal.json

```diff
@@ -0,0 +1,45 @@
+{
+  "description": "use over time report for a journal",
+  "type": "object",
+  "properties": {
+    "kbTitleId": {
```

Comment:
Rename to "kbId", rephrase the description to "KB identifier"

Reply:
Changed.